src/App.js

```diff
@@ -25,7 +26,7 @@ class App extends Component {
         },
         {
           id: 1,
-          title: "GDP",
+          title: "GDP/Capita",
           selected: false,
```

Comment:
Good change here definitely helps user get a better understanding of what our gdp data particularly references.